modules/itc/src/main/scala/lucuma/itc/search/syntax/Wavelength.scala

```diff
@@ -20,11 +20,10 @@ final class WavelengthOps(val self: Wavelength) extends AnyVal {
 
   /** Returns the sum of this wavelength and `other`, clipped at Wavelength.Max. */
   def +(other: Wavelength): Wavelength =
-    Some(self.toPicometers.value.value + other.toPicometers.value.value)
-      .filter(_ <= Wavelength.Max.toPicometers.value.value)
```

Comment:
The filter didn't do anything, because the max value of `toPicometers` is `Int.MaxValue`, and there can't be an Int bigger than that.